Wrong shot number:
[5, 4, 6]

Correct shot number:
[6]

Raw json:

{'hasError': True, 'errParameter': ['SECURE_HSTS_SECONDS'], 'reason': ["The property 'SECURE_HSTS_SECONDS' has the value '4000000000' which exceeds the range of an Integer."]}

{'hasError': True, 'errParameter': ['SECURE_HSTS_SECONDS'], 'reason': ["The property 'SECURE_HSTS_SECONDS' has the value '4000000000' which exceeds the range of an Integer."]}

{'hasError': True, 'errParameter': ['SIGNING_BACKEND'], 'reason': ["The property 'SIGNING_BACKEND' has the value 'django.core.signing.TimestampSigner' which is not within the accepted value {django.core.signing.Signer,django.core.signing.TimestampSigner}."]}

Final result:

There are 1 misconfiguration parameters in the input: SECURE_HSTS_SECONDS
[Ciri] Reason for SECURE_HSTS_SECONDS: The property 'SECURE_HSTS_SECONDS' has the value '4000000000' which exceeds the range of an Integer.
